app/assets/javascripts/application.js

```diff
@@ -3,7 +3,7 @@
 //= require jquery_ujs
 //= require jquery-ui
 //= require bootstrap-sprockets
-//= require bootstrap
+//////// require bootstrap
```

Comment:
Extra slashes makes it a comment. We need EITHER bootstap-sprockets or boortstrap but not both